tests/playlists.py

```diff
@@ -711,6 +711,7 @@
 #EXTINF:2.040,
 master2500_47226.ts
 #EXT-OATCLS-SCTE35:/DAlAAAAAAAAAP/wFAUAAAABf+//wpiQkv4ARKogAAEBAQAAQ6sodg==
+#EXT-X-ASSET:GENRE=CV,CAID=12345678,EPISODE="Episode%20Name%20Date",SEASON="Season%20Name%20and%20Number",SERIES="Series%2520Name"
```

Comment:
This is the example given [here](https://docs.aws.amazon.com/mediatailor/latest/ug/hls-ad-markers.html#hls-ad-markers-asset).